scss/modules/_typography-fonts.scss

```diff
@@ -25,3 +25,6 @@ $headingFont: 'Roboto Slab', 'serif';
  * }
  */
 $mainFont: 'Helvetica Neue', 'Helvetica', Arial, sans-serif;
+
+@import url("http://fonts.googleapis.com/css?family=Open+Sans:800");
+$headingEventFont: 'Open Sans', 'sans-serif';
```

Comment:
I'm not sure. Is there any reason why the default typography should be different in events than in other projects?